controlplane/modules/route/internal/discovery/bird/export.go

```diff
@@ -0,0 +1,140 @@
+package bird
+
+import (
+	"bufio"
+	"context"
+	"fmt"
+	"net"
+	"time"
+
+	"go.uber.org/zap"
+	"golang.org/x/sync/errgroup"
+
+	"github.com/yanet-platform/yanet2/controlplane/modules/route/internal/rib"
+)
+
+type RIBUpdater interface {
+	BulkUpdate([]*rib.Route) error
+}
+
+type exportSocket struct {
+	name    string
+	path    string
+	bufSize int
+}
+
+type Export struct {
+	sockets []exportSocket
+	ch      chan []rib.Route
+	cfg     *Config
+	updater RIBUpdater
+	log     *zap.SugaredLogger
+}
+
+func NewExportReader(cfg *Config, ribUpdater RIBUpdater, log *zap.SugaredLogger) *Export {
+	sockets := make([]exportSocket, 0, len(cfg.Sockets))
+	for _, s := range cfg.Sockets {
+		sockets = append(sockets, exportSocket{
+			name:    s.Name,
+			path:    s.Path,
+			bufSize: int(cfg.ParserBufSize.Bytes()),
+		})
+	}
+	return &Export{
+		sockets: sockets,
+		cfg:     cfg,
+		updater: ribUpdater,
+		log:     log,
+	}
+}
+
+func (m *Export) Run(ctx context.Context) error {
+	if !m.cfg.Enable {
+		m.log.Info("Bird export reader is disabled")
+		return nil
+	}
+
+	updates := make(chan *rib.Route, 100)
```

Comment:
Is `100` is a magic constant or it's some kind proved reasonable value?

Reply:
I think any value greater than zero will be enough here. I will change it to 10, but I don't like the idea of moving it to the config, as it would overwhelm the user without providing significant improvements to system stability.